FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -207,16 +205,11 @@ namespace edm {
       // 2) if it is a WorkerT<EDProducer>, add it to our list
       // 3) hand list to our delayed reader
       for (auto const& label : unusedLabels) {
```

Comment:
At this point, is `unusedLabels` actually an accurate variable name? Maybe `labelsForUnscheduled`?